os_net_config/cli.py

```diff
@@ -16,17 +16,15 @@
 
 
 import argparse
+import importlib
 import json
 import os
 import sys
 import yaml
 
+from os_net_config import NetConfig
 from os_net_config import common
 from os_net_config import dcb_config
```

Comment:
This needs to be done after importing common to avoid the pep8 error.

Reply:
done